src/main/java/at/hannibal2/skyhanni/config/features/skillprogress/SkillETADisplayConfig.java

```diff
@@ -0,0 +1,52 @@
+package at.hannibal2.skyhanni.config.features.skillprogress;
+
+import com.google.gson.annotations.Expose;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorBoolean;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorSlider;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorText;
+import io.github.moulberry.moulconfig.annotations.ConfigOption;
+import io.github.moulberry.moulconfig.observer.Property;
+
+public class SkillETADisplayConfig {
+
+    @Expose
+    @ConfigOption(name = "Enabled", desc = "Show a display of your current active skill\n" +
+        "with the XP/hour rate, ETA to the next level and current session time.")
+    @ConfigEditorBoolean
+    public Property<Boolean> enabled = Property.of(false);
+
+    @Expose
+    @ConfigOption(name = "Custom Goal", desc = "Choose a target level instead of showing progress to the next one.")
+    @ConfigEditorBoolean
+    public boolean enableCustomGoal = false;
```

Comment:
Use a command instead. we dont want to use the same target level for every skill. Also make custom goal compatible with skill progress and the progress bar.